### PR TITLE
F t32796 di plan beteiligung daten k3 weiterleiten

### PR DESCRIPTION
**Pull request by jalalifardemos**
**Ticket:** https://yaits.demos-deutschland.de/T32796

Set Migration for ProcedureMessage Table in relation with XBeteiligung's Entity

### How to review/test
check Migration File or run it

### Linked PRs (optional)

- {PR-number1} (https://github.com/demos-europe/demosplan-addon-xbeteiligung-async/pull/2)

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/DoctrineMigrations/2023/06/Version20230626135116.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function up(Schema $schema): void
+    {
+        $this->abortIfNotMysql();
+
+        $this->addSql('CREATE TABLE procedure_message (id CHAR(36) NOT NULL, procedure_id CHAR(36) NOT NULL, message LONGTEXT NOT NULL, created_date DATETIME DEFAULT CURRENT_TIMESTAMP NOT NULL, modification_date DATETIME DEFAULT CURRENT_TIMESTAMP NOT NULL, error TINYINT(1) DEFAULT false NOT NULL, deleted TINYINT(1) DEFAULT false NOT NULL, request_count INT DEFAULT 1 NOT NULL, UNIQUE INDEX UNIQ_E7F5DA961624BCD2 (procedure_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET UTF8 COLLATE `UTF8_unicode_ci` ENGINE = InnoDB');
```
Comment (mussbach) [inline demosplan/DemosPlanCoreBundle/DoctrineMigrations/2023/06/Version20230626135116.php:33]: Is `request_count INT DEFAULT 1` corrext? shouldn't it default to 0?

File: demosplan/DemosPlanCoreBundle/DoctrineMigrations/2023/06/Version20230626135116.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+use Doctrine\DBAL\Exception;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+class Version20230626135116 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'refs T:32796 create ProcedureMessage in relation with XBeteiligung ';
```
Comment (mussbach) [inline demosplan/DemosPlanCoreBundle/DoctrineMigrations/2023/06/Version20230626135116.php:23]:
```suggestion
        return 'refs T32796 create ProcedureMessage in relation with XBeteiligung ';
```

**Review by mussbach — APPROVED** [review]
lgtm, code review only

**Comment by jalalifardemos** [comment]
https://github.com/demos-europe/demosplan-core/pull/1664

**Pull request closed without merging**